### ISSUE TITLE
React site: extension progress

### ISSUE THREAD

**Issue by AprilSylph** [new-xkit/XKit#1802]
Tracking issue for which extensions need to be updated for the new dashboard.
Obsolete and joke extensions not listed. (Further reading: #1794, #1796)

- [ ] AccessKit
- [ ] Audio Downloader
- [ ] Audio+
- [ ] Auto Tagger
- [ ] Blacklist
- [ ] Blog Tracker
- [ ] Bookmarker (potentially impossible)
- [ ] Classic Notifications
- [ ] Classic Search
- [ ] CleanFeed
- [ ] Disable GIFs
- [ ] Don't Stretch Photosets
- [ ] Drafts+
- [ ] Editable Reblogs
- [ ] Enhanced Queue
- [ ] Find Blogs
- [ ] Header Options
- [ ] Hide Avatars
- [ ] Highlighter
- [ ] Iconified Notifications
- [ ] J/K Across Pages
- [ ] Limit People
- [ ] Mass Deleter
- [ ] Messaging Tweaks
- [ ] Mirror Button
- [ ] Mute!
- [ ] Mutual Checker
- [ ] No Recommended
- [ ] NotificationBlock
- [x] Old Blue
- [ ] Old Sidebar
- [ ] Old Stats
- [x] One-Click Postage
- [ ] One-Click Reply
- [ ] Open In Tabs
- [ ] Panorama
- [ ] Pokés
- [ ] Post Archiver
- [ ] Post Limit Checker
- [ ] PostBlock
- [ ] Profiler
- [ ] Quick Tags
- [ ] Read Posts
- [ ] Reblog Display Options
- [ ] Retags
- [ ] Scroll To Bottom
- [ ] Search Likes
- [ ] Separator
- [ ] Servant
- [ ] Shorten Posts
- [ ] Show Originals
- [ ] Show Picture Size
- [ ] Tab titles
- [ ] Tag Replacer
- [ ] Tag Tracking+
- [ ] Theme Editor / Themes / Themes+ (preview)
- [ ] Timestamps
- [ ] Tweaks
- [ ] Unreverse
- [ ] Vanilla Audio
- [ ] Vanilla Videos
- [ ] View My Tags
- [ ] XStats

No updates needed (yet):
- XInbox
- Activity+
- Auto Scroll
- Convert Links
- Find Inactives
- Glowing Follow
- Go-To-Dash
- Mass+
- Outbox
- Post Crushes
- Satsukimous
- XCloud
- XNeko

**Comment by hobinjk** [comment]
This is out of date and might be inaccurate, etc. etc. disclaimer disclaimer

Auto Tagger 7
Post editor and post types at least

Blog Tracker 3
frame run, simple post listener, sidebar

Classic Notifications 1
activity page scrape, bottom of new post buttons container

Classic Search 1
Actually easy

CleanFeed 7
Post listener, css

Disable GIFs 1
Post listener, easy css

Don't Stretch Photosets 1
Just css

Drafts+ 7
Css and post listener, mama mia

Enhanced Queue 3
Css, post id retrieval

Find Blogs 1
UMP

Header Options 1
Css

Highlighter 5
Post listener with tags, css

Iconified Notifications 5
Activity css, css generated by xkit?, more css

J/K Across Pages 1
keydown listener and post id

Limit People 3
Simplie-ish post listener, some post info

Mass Deleter 3
XKit.interface.get_posts, some post info

Messaging Tweaks 3 or 7
Css on messaging ui

Mirror Button 3
Post listener, simple post info

Mute! 7
Post listener, post types, some css

Mutual Checker 3
blog_listener, some css

NotificationBlock 7
Control button, post listener, activity page mod

Old Sidebar 3
css

Old Stats 3
css, light html mod

One-Click Reply 3
notification control buttonn, blogging as a concept

Post Archiver 7
iframe, css, post control, spicy post viewing?

Post Limit Checker 1
API + UI, should already work

PostBlock 3
Post listener, control button

Profiler 3
Post listener, control button, mild ui

Quick Tags 5
post editing, control button

Read Posts 3
Post listener, some css

Reblog Display Options 7
Post listener, post formatting

Retags 7
Pls no, obsolete-ish

Scroll To Bottom 1
Post listener, easy

Search Likes 5
Sidebar ui, post listener, some post data

Separator 1
Simple post listener

Servant 11
Everything

Shorten Posts 3
Post listener, post data, css

Show Originals 3
Sidebar control, post listener, post data

Show Picture Size 3
Post listener, some content modification

Tab titles 1
Barely anything

Tag Replacer 1
API

Tag Tracking+ 5
Sidebar control, post listener-ish, post data

Theme Editor / Themes / Themes+ (preview) 5
Fancy css mapping

Tweaks 7
So much css, post listeners

Unreverse 3
Tiny css, post listener

Vanilla Audio 1
Post listener, mild content edit

Vanilla Videos 1
Post listener, mild content edit

View My Tags 3
user data, kinda weird

XStats 5
sidebar control, blogging, api stuff

**Comment by inklesspen** [comment]
Why is bookmarker "(Arguably) Impossible to update"?

**Comment by AprilSylph** [comment]
@inklesspen the new dashboard's pagination doesn't work via post IDs / doesn't have unique URLs per page, so programmatically skipping to a point in time on the dashboard is impossible through that method

**Comment by inklesspen** [comment]
That's what I thought, but surely it can work through whatever means the Go-to-Dash extension has been fixed to work?

**Comment by AprilSylph** [comment]
the workaround implemented in Go-To-Dash isn't permanent and shouldn't be considered a "fix". Bookmarker could also implement the workaround, but the workaround is going to be patched out, probably soon

**Comment by Tails-155** [comment]
How far away does it look like the ETA is for quick tags? IS it one of the higher or lower priority things?

**Comment by AprilSylph** [comment]
@Tails-155 there is no ETA or official priority for any extensions.
however, Quick Tags is actively being worked on currently; see the Pull requests tab

**Comment by Tails-155** [comment]
> @Tails-155 there is no ETA or official priority for any extensions.
> however, Quick Tags is actively being worked on currently; see the Pull requests tab
Cool, cool. I definitely don't want to rush things, since I'd rather it work, and frankly I'm impressed it is already even partly compatible with a pure overhaul, I just know I use that extension a lot. XD

**Comment by chickenwithakeyboard** [comment]
Go-to-Dash isn't working at all anymore, even with the sidebar option.

**Comment by AprilSylph** [comment]
@chickenwithakeyboard this is due to a tumblr update that we're aware of and have a pending fix for; see #1920

**Comment by SupremeDevice** [comment]
It looks like Bookmarker could be updated to work with the new dash! After reloading my dashboard tonight it came up with a URL like https://www.tumblr.com/dashboard?max_post_id=(POSTIDHERE).
Unfortunately.... old bookmarks likely won't work. I exported my XKit data and tried some of the old IDs, and while I have been able to reach OLD (easily 4 years back!) posts, they don't quite seem to be the post I was looking for? (though some are really close!) But this may at least allow for new bookmarks to be made.

**Comment by inklesspen** [comment]
@SupremeDevice The general rule is that you have to add 1 to the post id, and then put _that_ in the URL. That's what works with the Go-to-Dash extension in PR #1936.

When the max_post_id thing first came out, trying to go to a really old post id would get a page full of "since you've been gone" recommended posts. but now it seems to work. I can go to [a four-year-old post](https://fialleril.tumblr.com/post/156337982051/why-mmfr-was-the-most-tasteful-action-movie-ive) using this method!

**Comment by SupremeDevice** [comment]
@inklesspen Ah, yep, that seems to work. For some reason my first guess was to lower the number and not increase it, but it seems on several old bookmarks that does work, and I assume the others that don't can be chalked up to general tumblr nonsense, or unfollowed/deleted blogs or posts. Really glad that bookmarks can be brought back again (but also that I now have a backup of them LOL)

**Comment by esotericDisciple** [comment]
Mirror Button Still Not Showing and therefore not Working

**Comment by AprilSylph** [comment]
Extensions listed but not checked here aren't impossible to update. Progress is simply slow, since this version of XKit is ill-equipped to handle the React site.

There's no definitive list of remaining legacy-powered pages, but the ones I can think of off the top of my head include:
- Inbox
- Settings
- Blog Members (this one may show Blog Tracker)
- Explore
- Themes

**Comment by citrusella** [comment]
I feel the strange need to note that, for now, reblogs that are done from a blog page open on a legacy dash (though that's not *precisely* helpful in most cases because actually *using* the legacy dash behind the reblog requires deleting the reblog modal and blue overlay nodes as well as changing the overflow for the page to a scrollable one).

TL;DR: There's another legacy-powered page but actually accessing/using it is hacky/weird. But it could be useful to some people.
